opensrp-chw/src/main/java/org/smartregister/chw/util/Utils.java

```diff
@@ -21,13 +24,32 @@
 public class Utils extends org.smartregister.chw.core.utils.Utils {
 
     public static void launchClientReferralActivity(Activity activity, List<ReferralTypeModel> referralTypeModels, String baseEntityId) {
+        if (activity instanceof AllClientsMemberProfileActivity){
+            ClientReferralActivity.isStartedFromAllClients = true;
+        }
         Bundle bundle = new Bundle();
         bundle.putString(Constants.ENTITY_ID, baseEntityId);
         bundle.setClassLoader(ReferralTypeModel.class.getClassLoader());
         bundle.putParcelableArrayList(Constants.REFERRAL_TYPES, (ArrayList<ReferralTypeModel>) referralTypeModels);
         activity.startActivity(new Intent(activity, ClientReferralActivity.class).putExtras(bundle));
     }
 
+    @NotNull
+    public static List<ReferralTypeModel> getCommonReferralTypes(Activity activity) {
+        List<ReferralTypeModel> referralTypeModels = new ArrayList<>();
+        if (BuildConfig.USE_UNIFIED_REFERRAL_APPROACH) {
+            referralTypeModels.add(new ReferralTypeModel(activity.getString(R.string.hiv_referral),
+                    org.smartregister.chw.util.Constants.JSON_FORM.getHivReferralForm()));
+
+            referralTypeModels.add(new ReferralTypeModel(activity.getString(R.string.tb_referral),
+                    org.smartregister.chw.util.Constants.JSON_FORM.getTbReferralForm()));
+
+            referralTypeModels.add(new ReferralTypeModel(activity.getString(R.string.gbv_referral),
+                    org.smartregister.chw.util.Constants.JSON_FORM.getGbvReferralForm()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.chw.util.Constants.JSON_FORM.getGbvReferralForm' due to existing same package import 'org.smartregister.chw.util.*'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426619)